client/state/reader/lists/selectors.js

```diff
@@ -63,9 +63,10 @@ export const getSubscribedLists = createSelector(
 		sortBy(
 			filter( state.reader.lists.items, ( item ) => {
 				// Is the user subscribed to this list?
-				return includes( state.reader.lists.subscribedLists, item.ID );
+				return state.reader.lists.subscribedLists.includes( item.ID );
 			} ),
-			'slug'
+			// Enable case-insensitive sort by title
+			( item ) => item.title?.toLowerCase()
```

Comment:
maybe a future PR, but it may be "better" to sort using a Collator (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator)

Reply:
Good call! I didn't know about those. Should probably switch the feed sorting to use one too.

@blowery do you know if it'll pick up the current locale by default, or do we need to provide one?

Reply:
(Going to 🚢 for now and pick this up in a followup PR.)